services/core-web/src/components/mine/DigitalPermitCredential/ViewDigitalPermitCredential.tsx

```diff
@@ -60,72 +38,51 @@ export const ViewDigitalPermitCredential: FC = () => {
   );
   const mine: IMine = useSelector((state) => getMineById(state, mineGuid));
 
-  const mineCommodityOptions: IMineCommodityOption[] = useSelector(getMineCommodityOptions);
-
-  const mineDisturbanceOptions: IMineDisturbanceOption[] = useSelector(getMineDisturbanceOptions);
-
   useEffect(() => {
     if (permitRecord) {
       dispatch(fetchCredentialConnections({ partyGuid: permitRecord.current_permittee_guid }));
     }
   }, [permitRecord]);
 
-  const getLatestIssueDate = () => {
-    const latestAmendment = permitRecord?.permit_amendments?.filter(
-      (a) => a.permit_amendment_status_code !== "DFT"
-    )[0];
-
-    return latestAmendment?.issue_date;
+  const openCredentialContentsModal = (event, cred_exch_id) => {
+    event.preventDefault();
+    dispatch(
+      openModal({
+        props: {
+          title: "Digital Credential Details",
+          partyGuid: minesActPermitIssuance[0].party_guid,
+          credExchId: cred_exch_id,
+          mine,
+        },
+        width: "50vw",
+        content: modalConfig.CREDENTIAL_CONTENT_MODAL,
+      })
+    );
   };
 
   const permitHistoryColumns = [
-    renderDateColumn("issue_date", "Issued"),
-    renderDateColumn("authorization_end_date", "Expiry"),
     {
-      key: "current_permittee_digital_wallet_connection_state",
+      title: "Credential Type",
+      render: () => "AnonCreds",
+    },
+    renderDateColumn("last_webhook_timestamp", "Last Updated"),
+    // renderTextColumn("cred_exch_state", "Status"),
```

Comment:
Will this commented line of code be used in future?

Reply:
Nope!  Good catch!